src/main/java/com/ericsson/ei/controller/FailedNotificationController.java

```diff
@@ -10,22 +10,22 @@
 
 
 /**
- * This class searches for missed notifications in the database.
+ * This class searches for failed notifications in the database.
  * (Generated with springmvc-raml-parser v.2.0.4)
  * 
  */
 @RestController
 @Validated
-@RequestMapping(value = "/queryMissedNotifications", produces = "application/json")
-public interface QueryMissedNotificationController {
+@RequestMapping(value = "/failedNotifications", produces = "application/json")
```

Comment:
the endpoint should follow the others spinal-case (with hyphen) instead of camelCase. change it to "/failed-notifications".

Reply:
done